SI/Sem.04/Pract.04/Task 04.cpp

```diff
@@ -0,0 +1,25 @@
+#include <iostream>
+
+int abs(int num)
+{
+    int result=(num >= 0) ? num :  -num;
+    return result;
+}
+
+double fabs(double num)
+{
+   int result = (num >= DBL_EPSILON) ? num : -num;
+    return result;
```

Comment:
Not working correctly. You are assigning a double to int so you are returning the int that will be converted to double